packages/markers/src/browser/problem/problem-widget.tsx

```diff
@@ -71,8 +76,13 @@ export class ProblemWidget extends TreeWidget {
         }
     }
 
+    protected updateGlobalSelection(): void {
+        this.selectionService.selection = TreeWidgetSelection.create(this);
+    }
+
     protected renderTree(model: TreeModel): React.ReactNode {
         if (MarkerRootNode.is(model.root) && model.root.children.length > 0) {
+            this.updateGlobalSelection();
```

Comment:
please don't call this method, you can only override it, `TreeWidget` know when is a proper timing

Reply:
Thanks for the explanation Anton. Now I can figure out how globalSelection works ;)